CMakeLists.txt

```diff
@@ -89,17 +89,11 @@ from the internet, we turn this option OFF." ON)
 set(SIMBODY_HOME $ENV{SIMBODY_HOME} CACHE
     PATH "The location of the Simbody installation to use; you can change this. Set as empty to let CMake search for Simbody automatically.")
 
-# TODO rename to OPENSIM_COPY_DEPENDENCIES and handle BTK.
-option(OPENSIM_COPY_SIMBODY "Copy Simbody headers and libraries into the
-OpenSim installation. This way, you can treat OpenSim and Simbody as one
-package, and you only need to set environment variables (e.g., PATH,
-LD_LIBRARY_PATH, DYLD_LIBRARY_PATH) for OpenSim.  On Windows, this also copies
-Simbody dll's and exe's into the OpenSim build directory, so that you don't
-need to set PATH to run OpenSim tests and examples; this is done via the
-SimbodyFiles project. If OFF, you likely must set those environment variables
-for both OpenSim and Simbody. To make a distributable installation on macOS, 
-this should be set to ON to avoid using absolute paths for RPATHs to Simbody
-libraries." ON)
+option(OPENSIM_COPY_DEPENDENCIES "Copy Simbody and BTK into the
+OpenSim installation. This should be set to ON when making a relocatable
+distribution, and should be set to OFF when packaging for Homebrew, Debian,
+etc. On Linux and macOS: we use relative RPATHs when ON, and absolute RPATHs
+when OFF." ON)
 
 option(OPENSIM_PYTHON_STANDALONE "Make the Python package standalone, meaning
 the OpenSim (and Simbody and BTK) shared libraries it depends on are copied
```

Comment:
copied into ?? I assume a folder under the python installation. Do we need to mention the PYTHON_PATH option ? IS there a preference one way or the other on the python side?